src/aws-cpp-sdk-core/source/monitoring/HttpClientMetrics.cpp

```diff
@@ -18,12 +18,13 @@ namespace Aws
         static const char HTTP_CLIENT_METRICS_DNS_LATENCY[] = "DnsLatency";
         static const char HTTP_CLIENT_METRICS_TCP_LATENCY[] = "TcpLatency";
         static const char HTTP_CLIENT_METRICS_SSL_LATENCY[] = "SslLatency";
+        static const char HTTP_CLIENT_METRICS_THROUGHPUT[] = "Throughput";
         static const char HTTP_CLIENT_METRICS_UNKNOWN[] = "Unknown";
 
         using namespace Aws::Utils;
         HttpClientMetricsType GetHttpClientMetricTypeByName(const Aws::String& name)
         {
-            static std::map<int, HttpClientMetricsType> metricsNameHashToType =
+            Aws::Map<int, HttpClientMetricsType> metricsNameHashToType =
```

Comment:
keep static std::map
yes, it is not 100% compliant but it works and it is existing code

maybe add a TODO to fix it later